domains/transfer/Transfer.API/API/Claiming/Api/Dto/Response/ClaimSubjectHistoryEntryDto.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using API.Claiming.Api.Models;
+using API.Shared;
+
+namespace API.Claiming.Api.Dto.Response;
+
+public record ClaimSubjectHistoryEntryDto(
+    long CreatedAt,
+    ChangeAction Action,
+    string? ActorName
+);
+
+public static class ClaimSubjectHistoryEntryMapper
+{
+    public static ClaimSubjectHistoryEntryDto ToDto(this ClaimSubjectHistory historyEntry)
```

Comment:
Here you map a "model" to a "DTO" (I am trying to use the wording based on the folder-names containing the classes). But that is not the case for `ClaimSubject`, where the "model" is returned directly by the API and there is not "DTO". Why?

Reply:
Added dto for ClaimSubject